testdata/multi_module.asn

```diff
@@ -17,4 +17,9 @@ AnotherSequence ::= SEQUENCE {
     second BOOLEAN
 }
 
+YetAnotherSequence ::= SEQUENCE {
+    first [0] Module1.OneSequence,
```

Comment:
Good idea, thanks -- this is good enough for now at least, I'll have to think about how to test multiple Python modules.